src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -283,3 +294,31 @@ def find_version_info_in_manifest_file(
             patch_name = version_info[patch_field]
 
     return version_name, patch_name
+
+
+def project_path_to_deploy_path(
+    project_path: Path, mapped_files: ArtifactDeploymentMap
```

Comment:
Nit: is this `project_path` or `source_path`? `project_path` can be misleading to only mean the path to the project, i.e. root of the project.

Reply:
Agreed, I think `source_path` is a better name.